tensorly/decomposition/_parafac2.py

```diff
@@ -17,8 +18,57 @@
 #          Yngve Mardal Moe
 
 
+def _update_imputed(tensor_slices, mask, decomposition=None, method="mode-2"):
+    """
+    Update missing values of tensor slices according to method.
+
+    Parameters
+    ----------
+    tensor_slices : Iterable of ndarray
+    mask : ndarray
+        array of booleans with the same shape as tensor should be 0 where the values are
+        missing and 1 everywhere else.
+    decomposition : Parafac2Tensor, optional
+    method : string
+        One of 'mode-2' or 'factors'. 'mode-2' updated imputed values according to mean of each mode-2 slice. If 'factors' is chosen, update imputationa according to reconstruction given from 'decomposition'.
```

Comment:
I think it would be nice if this docstring also stated that "mode-2" was only used for initialisation and not used during model fitting.

Reply:
Also addressed in a20b562526de762ff7e1f6296d92136e8348dd5b.